src/main/php/com/mongodb/result/Cursor.class.php

```diff
@@ -6,16 +6,18 @@
 use util\Objects;
 
 class Cursor implements Value, IteratorAggregate {
-  private $proto, $current;
+  private $proto, $session, $current;
 
   /**
    * Creates a new cursor
    *
-   * @param com.mongodb.io.Protocol $proto
-   * @param [:var] $current
+   * @param  com.mongodb.io.Protocol $proto
+   * @param  ?com.mongodb.Session $session
+   * @param  [:var] $current
    */
-  public function __construct($proto, $current= []) { 
+  public function __construct($proto, $session, $current= []) { 
```

Comment:
`$current` cannot be empty, getIterator() expects it to contain "ns" and "id" members.